boilerplate/frontend/supertokens-react-custom/config/all_auth.tsx

```diff
@@ -0,0 +1,106 @@
+import EmailPassword from "supertokens-auth-react/recipe/emailpassword";
```

Comment:
please see our docs for custom UI. We use the supertokens-web-js sdk. Not the supertokens-auth-react one.

Reply:
Updated the imports to use web-js SDK